src/actions/receive-profile.js

```diff
@@ -450,14 +450,14 @@ async function _fetchProfile({ url, onTemporaryError }: FetchProfileArgs) {
     }
 
     // case 2: unrecoverable error.
-    if (response.status !== 404) {
+    if (response.status !== 403) {
```

Comment:
Could we check for a range of values here to cover our bases?

Reply:
I'm not sure we'd want this. We want to retry _only_ when the upload is happening but not finished. I always felt `404` wasn't an appropriate response and I'm quite OK with google changing it. Now we could return a better error message if there is a 404 -- which likely means that the user used a wrong hash.

Reply:
Works for me, thanks.